device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/InternalClient.java

```diff
@@ -887,6 +887,7 @@ void setOption_SetCertificatePath(Object value)
     {
         if (value != null)
         {
+            log.info("Setting path to trusted certificate");
```

Comment:
Not logging the path being set because it has no value for our debugging purposes. If the path doesn't point at a valid certificate, then the SDK will throw an exception anyways.